test/base.jl

```diff
@@ -6,4 +6,13 @@
     reset!(env)
     run(policy, env)
     @test get_terminal(env)
+
+    discrete_env =  env |> ActionTransformedEnv(
+        a -> get_actions(env)[a];  # action index to action
+        mapping = x -> Dict(x => i for (i, a) in enumerate(get_actions(env)))[x] # arbitrary vector to DiscreteSpace
```

Comment:
Should it be `Dict(a => i for (i, a) in enumerate(get_actions(env)))` instead of `Dict(x => i for (i, a) in enumerate(get_actions(env)))`

Reply:
👍

Reply:
Should I file a PR?

Reply:
Yes please.

Reply:
👍🏽